.rubocop.yml

```diff
@@ -64,6 +64,9 @@ RSpec/VariableName:
 Style/GuardClause:
   Enabled: false
 
+Style/IfUnlessModifier:
```

Comment:
Why are we changing this rule? It seems unrelated to the work.

Reply:
This pull request is still in draft. This was added due to a change in rubocop requirements but not be needed since rubocop was updated.